assignments/pig-latin/code.js

```diff
@@ -31,3 +31,12 @@
 //
 // Feel free to play around in the REPL to make sure you understand how this
 // operator works.
+
+const pigLatin = (s) => {
+  var o = (s.search(/[aeiou]/))
```

Comment:
This is fine but you don't need the extra `()`s around the expression. And `o` is maybe not the best variable name just because it looks like a `0`. Finally, don`t use `var`; it's old fashioned and has some subtle technical problems. For a case like this where you don't need to change the value use `const`. If you do need a variable that you will reassign, use `let`.

All together, you should change this to something like:

```suggestion
  let i = s.search(/[aeiou]/);
```